packaging/bin/engine-backup.sh.in

```diff
@@ -173,7 +173,6 @@ BACKUP_PATHS="/etc/ovirt-engine
 # Add /var/lib/ovirt-engine except a few
 VAR_LIB_OVIRT_ENGINE_EXCLUSIONS="/var/lib/ovirt-engine/backups
 /var/lib/ovirt-engine/jboss_runtime
-/var/lib/ovirt-engine/ansible-runner-service.log
```

Comment:
Why? See my previous reply. Thanks.

Reply:
I posted it the same time you posted your comment, 
anyway I just added it back :)

Reply:
Thanks :-)